docs/README.md

```diff
@@ -17,3 +17,4 @@
 - [Recover from gatekeeper outage](gds-supported-platform/recover-gatekeeper-outage.md)
 - [Sealing Secrets](gds-supported-platform/sealing-secrets.md)
 - [Set up monitoring and alerting with Prometheus and Alertmanager](gds-supported-platform/prometheus_alert_manager_grafana.md)
+- [Updating EKS](gds-supported-platform/prometheus_alert_manager_grafana.md)
```

Comment:
Updating EKS and prometheus alert manager grafana do not appear to be related to each other.

Reply:
Yeah, found these and a couple of issues @adityapahuja thanks, not thinking right today. All corrected.